clang/include/clang/Basic/Cuda.h

```diff
@@ -52,7 +52,7 @@ const char *CudaVersionToString(CudaVersion V);
 // Input is "Major.Minor"
 CudaVersion CudaStringToVersion(const llvm::Twine &S);
 
-enum class CudaArch {
+enum class GpuArch {
```

Comment:
Probably should call this OffloadArch to match --offload-arch

Reply:
Good idea, let me rename it real quick.

Reply:
Done.

Reply:
I've folded the commits to keep it clean.